src/apps/chifra/pkg/tslib/tslib.go

```diff
@@ -0,0 +1,143 @@
+package tslibPkg
+
+import (
+	"encoding/binary"
+	"errors"
+	"os"
+	"sort"
+	"strings"
+	"time"
+)
+
+const tsPath = "/Users/jrush/Development/trueblocks-index/wild.0/unchained/ts.bin"
```

Comment:
This path shouldn't be hardcoded, I guess

Reply:
Fixed.